fastify.js

```diff
@@ -707,3 +707,11 @@ function loadVersion () {
 module.exports = fastify
 module.exports.fastify = fastify
 module.exports.default = fastify
+
+try {
+  const dc = require('diagnostics_channel')
+  dc.channel('fastify:loaded').publish(fastify)
```

Comment:
Since https://github.com/fastify/fastify-diagnostics-channel already exists, we should at least use the same syntax, ie: `fastify.loaded`.

Reply:
Yeah I'm fine with using a `.` here.

Reply:
I would prefer if we checked with `hasSubscribers`.

Reply:
Why?

Reply:
no need!